amr-wind/core/SimTime.cpp

```diff
@@ -201,15 +257,23 @@ bool SimTime::do_regrid() const
 bool SimTime::write_plot_file() const
 {
     return (
-        (m_plt_interval > 0) &&
-        ((m_time_index - m_plt_start_index) % m_plt_interval == 0));
+        ((m_plt_interval > 0) &&
+         ((m_time_index - m_plt_start_index) % m_plt_interval == 0)) ||
```

Comment:
Is it worthwhile adding a check to make sure that both conditions are not being activated in the same simulation? Are there scenarios that we foresee otherwise?